lib/nostrum/api.ex

```diff
@@ -276,86 +290,155 @@ defmodule Nostrum.Api do
   end
 
   @doc ~S"""
-  Create a rection for a message.
+  Creates a reaction for a message.
 
-  Creates a reaction using an `emoji` for the message specified by `message_id` and
-  `channel_id`. `emoji` can be a `Nostrum.Struct.Emoji.custom_emoji.t`, a
-  base 16 unicode emoji string, or a uri encoded string.
+  This endpoint requires the `VIEW_CHANNEL` and `READ_MESSAGE_HISTORY` 
+  permissions. Additionally, if nobody else has reacted to the message with 
+  the `emoji`, this endpoint requires the `ADD_REACTIONS` permission. It 
+  fires the `MESSAGE_REACTION_ADD` event.
+
+  If successful, returns `{:ok}`. Otherwise, returns `t:Nostrum.Api.error/0`.
+
+  ## Examples
 
-  **Example**
   ```Elixir
+  # Using a Nostrum.Struct.Emoji.
+  emoji = %Nostrum.Struct.Emoji{id: 43819043108, name: "foxbot"}
+  Nostrum.Api.create_reaction(123123123123, 321321321321, emoji)
+
+  # Using a base 16 emoji string.
   Nostrum.Api.create_reaction(123123123123, 321321321321, "\xF0\x9F\x98\x81")
+
+  # Using a URI encoded emoji string.
   Nostrum.Api.create_reaction(123123123123, 321321321321, URI.encode("\u2b50"))
```

Comment:
I've made encoding the URL the default behavior, so this is no longer necessary (and actually wasn't in the first place).

Reply:
Fixed in latest commit.